<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueueForm</class>
 <widget class="QWidget" name="QueueForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Queue</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: &quot;Yu Gothic&quot;;
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="titleLine">
     <item>
      <widget class="QLabel" name="icon">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons.qrc">:/tank.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="title">
       <property name="styleSheet">
        <string notr="true">font-size: 20pt;</string>
       </property>
       <property name="text">
        <string>Panzer assault</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="statusLine">
     <item>
      <widget class="QLabel" name="status">
       <property name="styleSheet">
        <string notr="true">font-size: 9pt;</string>
       </property>
       <property name="text">
        <string>You are in queue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="queueTime">
       <property name="styleSheet">
        <string notr="true">font-size: 9pt;</string>
       </property>
       <property name="text">
        <string>00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="queueInfo">
     <item>
      <widget class="QLabel" name="players">
       <property name="styleSheet">
        <string notr="true">font-size: 9pt;</string>
       </property>
       <property name="text">
        <string>Players in queue:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="playersNumber">
       <property name="styleSheet">
        <string notr="true">font-size: 9pt;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="machineInfo">
     <item>
      <widget class="QLabel" name="select">
       <property name="styleSheet">
        <string notr="true">font-size: 9pt;</string>
       </property>
       <property name="text">
        <string>Selected machine:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="machineName">
       <property name="styleSheet">
        <string notr="true">font-weight: bold; font-size: 9pt;</string>
       </property>
       <property name="text">
        <string>MS-1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="machineLevel">
       <property name="styleSheet">
        <string notr="true">font-size: 9pt;</string>
       </property>
       <property name="text">
        <string>1 level</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="menuLine">
     <item>
      <widget class="QPushButton" name="buttonLeave">
       <property name="styleSheet">
        <string notr="true">font-size: 9pt;</string>
       </property>
       <property name="text">
        <string>Leave queue</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonTryAlone">
       <property name="styleSheet">
        <string notr="true">font-size: 9pt;</string>
       </property>
       <property name="text">
        <string>Try alone</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
